src/Analysis/Ast/Test/ImportTests.cs

```diff
@@ -223,7 +223,7 @@ import sys
             analysis.Should().HaveFunction("print");
         }
 
-        [TestMethod, Priority(0)]
+        //[TestMethod, Priority(0)]
```

Comment:
Can you ignore instead of comment?

Reply:
I am contemplating if we should remove this completely (and check for `Any`) in the stub merge. After all, if stub says `Any` so be it, even if from sources we decided it is, say, `int`.